src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -1,52 +1,55 @@
 <br>
```

Comment:
This `<br>` is unnecessary, can take the chance to remove it

Reply:
Removed